v21.2/as-of-system-time.md

```diff
@@ -33,8 +33,8 @@ Format | Notes
 negative [`INTERVAL`](interval.html) | Added to `statement_timestamp()`, and thus must be negative.
 [`STRING`](string.html) | A [`TIMESTAMP`](timestamp.html), [`INT`](int.html) of nanoseconds, or negative [`INTERVAL`](interval.html).
 `follower_read_timestamp()`| A [function](functions-and-operators.html) that returns the [`TIMESTAMP`](timestamp.html) `statement_timestamp() - 4.8s`. Using this function will set the time as close as possible to the present time while remaining safe for [exact staleness follower reads](follower-reads.html#exact-staleness-reads).
-`with_min_timestamp(TIMESTAMPTZ, [nearest_only])` | <span class="version-tag">New in v21.2:</span> Defines a minimum [timestamp](timestamp.html) at which to perform the [bounded staleness read](follower-reads.html#bounded-staleness-reads). The actual timestamp of the read may be equal to or later than the provided timestamp, but cannot be before the provided timestamp. This is useful to request a read from nearby followers, if possible, while enforcing causality between an operation at some point in time and any dependent reads. This function accepts an optional `nearest_only` argument that will error if the reads cannot be serviced from a nearby replica.
-`with_max_staleness(INTERVAL, [nearest_only])` | <span class="version-tag">New in v21.2:</span> Defines a maximum staleness interval with which to perform the [bounded staleness read](follower-reads.html#bounded-staleness-reads). The timestamp of the read can be at most this stale with respect to the current time. This is useful to request a read from nearby followers, if possible, while placing some limit on how stale results can be. Note that `with_max_staleness(INTERVAL)` is equivalent to `with_min_timestamp(now() - INTERVAL)`. This function accepts an optional `nearest_only` argument that will error if the reads cannot be serviced from a nearby replica.
+`with_min_timestamp(TIMESTAMPTZ, [nearest_only])` | <span class="version-tag">New in v21.2:</span> Defines a minimum [timestamp](timestamp.html) at which to perform the [bounded staleness read](follower-reads.html#bounded-staleness-reads). The actual timestamp of the read may be equal to or later than the provided timestamp, but cannot be before the provided timestamp. This is useful to request a read from nearby followers, if possible, while enforcing causality between an operation at some point in time and any dependent reads. This function accepts an optional `nearest_only` argument that will error if the reads cannot be serviced from a nearby replica. If a follower read can't be served when `nearest_only` is set to `true` and a schema change is made, CockroachDB will retry the read from before the schema change was made so long as the schema change occurred after the specified `TIMESTAMPTZ`.
```

Comment:
i'm trying to think of this makes the most sense to read from an external user.

Maybe `The query may fetch a result from an older schema if a read from the current cannot be satisified with a follower read.`?

Reply:
I'm happy to omit the detail or use your suggestion. Your call.

Reply:
Yeah let's omit it. The commit in actuality fixes a previous limitation related to schema changes. Now it's an internal detail that doesn't matter too much.

Reply:
+1 on omiting the detail. I think it will only serve to confuse readers.

Reply:
@otan @nvanbenschoten I omitted it from follower-reads and as-of-system-time, since it's the same message across all of it. Is the bit in transactions still good?